nupic/frameworks/opf/clamodel.py

```diff
@@ -441,6 +443,21 @@ def _getSensorInputRecord(self, inputRecord):
                        sequenceReset=resetOut,
                        category=inputRecordCategory)
 
+  def _getClassifierInputRecord(self, inputRecord):
```

Comment:
It is used for calculating the likelihood. To calculate LL, we need to know the probability of the true input, and that involves knowing the bucket index of the input from the classifier encoder. This method returned that information and include it to the model result